### PR TITLE
[WIP] adds admin martial art

### PR DESCRIPTION
**Pull request by hornygranny**
please suggest more overpowered/silly techniques

### CODE REVIEW

File: code/datums/martial.dm [hunk 1]
```diff
@@ -577,3 +577,141 @@
...
+
+
+/datum/martial_art/admin_fu/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	wristFlip(A,D)
+	return ..()
+
+/mob/living/carbon/human/proc/admin_fu_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of the Sleeping Carp clan."
+	set category = "Sleeping Carp"
```
Comment (Iamgoofball) [inline code/datums/martial.dm]: >sleeping carp

File: code/datums/martial.dm [hunk 2]
```diff
@@ -577,3 +577,141 @@
...
+		D.apply_damage(100, BRUTE, "chest")
+		playsound(get_turf(D), 'sound/weapons/punch1.ogg', 75, 1, -1)
+		return 1
+
+/datum/martial_art/admin_fu/proc/neckSnap(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D)
+	D.visible_message("<span class='warning'>[A] snaps [D]'s neck!</span>", \
+						 "<span class='userdanger'>[A] snaps on your neck!</span>")
+	D.apply_damage(400, BRUTE)
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 75, 1, -1)
```
Comment (PKPenguin321) [inline code/datums/martial.dm]: neck stomp should do 400 brute, neck snap should be the outright instakill

**Comment by PKPenguin321** [comment]
not silly enough, needs at least one move that gibs. plasma fist has gibbing

**Comment by PKPenguin321** [comment]
also there needs to be a thunder punch that sends opponents flying and shocks them and does a ton of burn

**Comment by TrustyGun** [comment]
add the pelvic pulveriser that stuns the target for an absolutely ridiculous amount of time, and turns them gender neutral

**Comment by TheRealGLH** [comment]
This badly needs a technique where you quickly jab your opponents pressure points, damaging their body from the inside and causing it to explode violently.
https://www.youtube.com/watch?v=YSgpU70MZno

**Comment by Menshin** [comment]
>not silly enough, needs at least one move that gibs

Seems like hg forgot this basic admin_fu move : https://www.youtube.com/watch?v=YSgpU70MZno .

**Comment by Cheridan** [comment]
The first thing that I thought of when i imagined "admin martial arts" was a move that mutes the victim. 
a move that turns the victim into something stupid, like an animated toilet.

**Comment by Core0verload** [comment]
There are already enough scroll items cluttering the items list. Maybe make them into a single type with subtypes?

**Comment by AlexanderUlanH** [comment]
Polymorphism is for nerds
That's also we have do_attack_animation in the code for every martial arts attack.

File: code/datums/martial.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/datums/martial.dm]: >snaps on your neck

**Comment by KazeEspada** [comment]
Roundhouse kick them so hard it gives them a ban message like the ban hammer, the bwoink sound, and kicks their client.

**Comment by RemieRichards** [comment]
>kicks their client
>roundhouse kick
>kicks
>roundhouse
>KICKS
>KICK

**Comment by Cheridan** [comment]
>it's a hg work in progress PR

**Pull request closed without merging**